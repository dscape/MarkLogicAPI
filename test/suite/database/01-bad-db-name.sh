#!/bin/sh
curl -sX PUT localhost:6984/TestShouldFail -d ""