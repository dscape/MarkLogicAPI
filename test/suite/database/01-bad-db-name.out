{"error":"illegal_database_name","reason":"Only lowercase characters (a-z), digits (0-9), and any of the characters _, and - are allowed. Must begin with a letter."}